elfi/client.py

```diff
@@ -70,37 +71,79 @@ def total(self):
 
     @property
     def num_ready(self):
-        return self.total - len(self.pending_indices)
+        return self.total - self.num_pending
```

Comment:
These could be just n_ready and n_pending to keep the style we've had elsewhere. 

For once I'm the one suggesting a shorter name... :D